testing/tests/DevExpress.ui.widgets.scheduler/common.tests.js

```diff
@@ -330,77 +319,131 @@ QUnit.testStart(function() {
         this.instance.option({
             dataSource: [{
-                startDate: new Date(2015, 4, 24, 9, 10),
-                endDate: new Date(2015, 4, 24, 11, 1),
-                recurrenceRule: "FREQ=DAILY;COUNT=2",
+                startDate: new Date(2015, 4, 24, 9),
+                endDate: new Date(2015, 4, 24, 11),
                 allDay: true,
+                recurrenceRule: "FREQ=DAILY;COUNT=3",
                 text: "Task 2"
             }],
             height: 600,
             currentDate: new Date(2015, 4, 24),
-            displayedAppointmentDataField: null,
             currentView: "month",
             views: ["month"],
-            appointmentTemplate: function(data) {
-                assert.strictEqual(data.Field, undefined);
+            appointmentTooltipTemplate: function(data, index, targetedAppointmentData) {
+                assert.deepEqual(targetedAppointmentData, {
+                    allDay: true,
+                    endDate: new Date(2015, 4, 25, 11),
+                    recurrenceRule: "FREQ=DAILY;COUNT=3",
+                    startDate: new Date(2015, 4, 25, 9),
+                    text: "Task 2"
+                });
             }
         });
+
+        $(this.instance.$element().find(".dx-scheduler-appointment").eq(1)).trigger("dxclick");
+        this.clock.tick(300);
     });
 
-    QUnit.test("appointmentTooltipTemplate rendering args should be correct if displayedAppointmentDataField is specified", function(assert) {
+    QUnit.test("the targetedAppointmentData parameter appends to arguments of the appointment tooltip template for a non-recurrence rule", function(assert) {
         this.instance.option({
             dataSource: [{
-                startDate: new Date(2015, 4, 24, 9, 10),
-                endDate: new Date(2015, 4, 24, 11, 1),
-                allDay: true,
-                recurrenceRule: "FREQ=DAILY;COUNT=2",
-                text: "Task 2"
+                startDate: new Date(2015, 4, 24, 9),
+                endDate: new Date(2015, 4, 24, 11),
+                text: "Task 1"
             }],
             height: 600,
             currentDate: new Date(2015, 4, 24),
-            displayedAppointmentDataField: "Field",
             currentView: "month",
             views: ["month"],
-            appointmentTooltipTemplate: function(data) {
-                assert.deepEqual(data.Field, {
-                    startDate: new Date(2015, 4, 25, 9, 10),
-                    endDate: new Date(2015, 4, 25, 11, 1),
-                    allDay: true,
-                    recurrenceRule: "FREQ=DAILY;COUNT=2",
-                    text: "Task 2"
-                }, "occurence args is correct");
+            appointmentTooltipTemplate: function(data, index, targetedAppointmentData) {
+                assert.deepEqual(data, targetedAppointmentData);
```

Comment:
The same objects are passed by two arguments? Can they be different like onAppointmentRendered(e.appointmentData and e.targetedAppointmentData)?